WebContent/WEB-INF/tags/page.tag

```diff
@@ -131,6 +144,99 @@
+                        document.getElementById("__header__alarmLevelText").innerHTML = response.alarmlevel;
+                        setAlarmLevelImg(alarmLevel, "__header__alarmLevelImg");
+                        setAlarmLevelText(alarmLevel, "__header__alarmLevelText");
+                        document.getElementById("__header__alarmLevelDiv").style.visibility='visible';
+                        document.getElementById("__header__alarmLevelImg").style.visibility='visible';
+                    }
+                    else {
+                        document.getElementById("__header__alarmLevelText").innerHTML = "";
+                        document.getElementById("__header__alarmLevelImg").style.visibility='hidden';
+                        document.getElementById("__header__alarmLevelDiv").style.visibility='hidden';
+                    }
+                })
+                stompClient.send("/app/alarmLevel", {priority: 1}, "STOMP - gimme my alarmLevel");
+            } );
+            stompClient.send("/app/alarmLevel", {priority: 9}, "STOMP");
+        };
+
+        function connect(url, headers, errorCallback, connectCallback) {
+            var socket = new SockJS(url);
+            var stompClient = Stomp.over(socket);
+            stompClient.heartbeat.outgoing = 20000;
+            stompClient.heartbeat.incoming = 0;
+            stompClient.debug = null;
+            stompClient.connect(headers, connectCallback, errorCallback);
+            return stompClient;
+        }
+
+        function disconnect() {
+            if(stompClient != null) {
+                console.log("Disconnecting...");
+                stompClient.disconnect(function() {
+                    console.log("Disconnected");
+                    stompClient = null;
+                });
+            }
+        }
+
+        function onloadHandler() {
+           var location = window.location;
+           var appName = location.pathname.split("/")[1];
+           var myLocation = location.origin + "/" + appName+ "/";
+           stompClient = connect(myLocation + 'ws-scada/alarmLevel', {}, errorCallback, connectCallback);
+        }
+
+        function setAlarmLevelText(alarmLevel, textNode) {
+            textNode = document.getElementById(textNode);
+            if (alarmLevel == 0)
+                textNode.innerHTML = "";
+            else if (alarmLevel == 1)
+                textNode.innerHTML = "Info";
+            else if (alarmLevel == 2)
+                textNode.innerHTML = "Urgent";
+            else if (alarmLevel == 3)
+                textNode.innerHTML = "Critical";
+            else if (alarmLevel == 4)
+                textNode.innerHTML = "Life Safety";
+            else
+                textNode.innerHTML = "Unknown: "+ alarmLevel;
+        }
+
+        function setAlarmLevelImg(alarmLevel, imgNode) {
+            if (alarmLevel == 0)
+                updateImg(imgNode, "images/flag_green.png", "undef", false, "none");
```

Comment:
this string "undef" can be change to "Green Flag" string.

This argument is the **alt** property of the image so we should try to describe here what is visible on the image for the screen-readers.